### DISCUSSION TITLE
LINQ C# expressions

### DISCUSSION THREAD

**Discussion by gsicurtis** [codewars/codewars.com#3144] [Q&A] [answered]
Is there a way to get CodeWars to recognize C# LINQ expressions. When I use code that works in Visual Studio, such as, int[].Sum() or bool[].Where(x => x), CodeWars tells me that there is no function named Sum or Where.

**Comment by hobovsky** [comment, marked as answer]
Codewars supports Linq and functions like `Sum` and `Where`. To use them, you need to add appropriate `using` to your solution, specifically `using System.Linq;`

**Reply by gsicurtis** [reply]
Perfect, thank you!
